Apps/AdminWebClient/src/ClientApp/src/services/interfaces.ts

```diff
@@ -47,7 +47,8 @@ export interface IDashboardService {
 
 export interface ICommunicationService {
   initialize(http: IHttpDelegate): void;
-  add(communication: Communication): Promise<void>;
+    add(communication: Communication): Promise<void>;
+    getBannerList(): Promise<Communication[]>;
```

Comment:
Looks like your spacing might be slightly off - try configuring your editor/IDE to match the spacing of the repository for formatting purposes

Reply:
good catch! Thanks @sam-warren. I'm using VS2019, I will need to compare my VS editor's settings with @sslaws or someone who is using VS2019 later to make sure I have the right settings for this.

Reply:
This should have been fixed with my most recent commit today.